tensorflow_addons/image/__init__.py

```diff
@@ -21,6 +21,7 @@
 from tensorflow_addons.image.cutout_ops import cutout
 from tensorflow_addons.image.dense_image_warp import dense_image_warp
 from tensorflow_addons.image.distance_transform import euclidean_dist_transform
+from tensorflow_addons.image.filters import gaussian_filter2d
```

Comment:
Move this imports to the end. Alphabetically.

Reply:
Can you please elaborate?
Earlier I was asked to put alpahbetically(at the present position).
See here-https://github.com/tensorflow/addons/pull/1450#discussion_r412414320

Reply:
I didn't mean at the bottom. Sorry for incomplete comment.  IMO, it should be like this:
```python
....

from tensorflow_addons.image.dense_image_warp import interpolate_bilinear
from tensorflow_addons.image.interpolate_spline import interpolate_spline
from tensorflow_addons.image.filters import gaussian_filter2d  ## ideally it should be placed here 
from tensorflow_addons.image.filters import mean_filter2d
from tensorflow_addons.image.filters import median_filter2d

....



```

Reply:
@AakashKumarNain  Done.